index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
 implementing the rule above. This can ensure proper term identification,
 typing, and order, when a JSON document is processed as JSON-LD.
           </p>
+          <p>
+However, the intention of the issuer is not preserved, unless the <code>@context</code>
+is applied, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>,
+and their exact RDF types, and node and edge structure is understood by the verifier.
```

Comment:
```suggestion
However, the intention of the issuer is not preserved, unless the rules expressed
in the <code>@context</code> are followed, and the resulting <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>, and their exact RDF types, and node and edge structure is understood by the verifier. By following the rules laid out in this section, applications can be hard-coded against
specific contexts, just as they are commonly hard-coded to consume other
application-specific JSON formats.
```

Reply:
"applications can be hard-coded against specific contexts, just as they are commonly hard-coded to consume other application-specific JSON formats."

^ can you update this to account for when the contexts change, and how that impacts what the verifier will understand  based on if the verifier is hard coded or is consuming a context that has errata published?

Reply:
@OR13, thoughts?

Reply:
I'm accepting this suggestion, but its misleading in the case that a hard coded context does not match one for which errata has been published.